### DISCUSSION TITLE
Multiple domains questions

### DISCUSSION THREAD

**Discussion by PrabhuShanmughapriyan** [postalserver/postal#2730] [Help with using Postal]
Hi,

I have 2 domains verified in mailserver

example.com
testing.com

When i send emails using testing.com, I still see the headers are using example.com for message ID.. any reason for that?
<img width="699" alt="Screenshot 2023-12-21 at 11 31 53 AM" src="https://github.com/postalserver/postal/assets/88906038/02ab106e-b12d-40fb-a188-487d7e60832c">
You can for example, the domain i am creating is called somedomain.com, but SPF and few other DNS entries show for different domain

**Comment by willpower232** [comment]
I'm presuming you saying example.com is your own reference and not what is in the config file by default.

If you're using SMTP (or the raw API) then whatever is constructing the email message would be to blame for including the slightly wrong reference.

The domains from the config file need to refer to the Postal server itself, as an aside, it looks like your DNS records are missing so if you didn't already know then perhaps you have not set up the DNS or the config file correctly.

In your screenshot, you have entered yoroconnect.com as the domains in the config file so those are ones which need to refer to Postal itself, i.e. should have the IP address of your server.

somedomains.com can then refer to the domains from the config file as shown in your screenshot so that you don't need to have the IP address of your Postal server everywhere.

Hope this makes sense, you could try sending emails to https://mail-tester.com to confirm that your DNS works and is trusted even if Postal is confused.
